services/icecast.nix

```diff
@@ -0,0 +1,62 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  common = import ../common/variables.nix;
+  tld = common.tld;
+  cfg = config.services.icecast;
+  admin-secret = "/var/secrets/icecast-admin.secret";
+  source-secret = "/var/secrets/icecast-source.secret";
+  relay-secret = "/var/secrets/icecast-relay.secret";
+in {
+  users.users.icecast = {
+    description = "Service user for icecast";
+    isSystemUser = true;
+    group = "icecast";
+    shell = "/dev/null";
+    home = "/dev/null";
+  };
+
+  services.icecast = {
+    enable = true;
+    hostname = "dcufm.${tld}";
+    user = "icecast";
+    admin = {
+      password = "${lib.fileContents admin-secret}";
+    };
+    listen = {
+      port = 8002;
+    };
+    logDir = "/var/log/icecast/";
```

Comment:
not anything to fix but how does logging to syslog work with this?

Reply:
I'm not actually sure on that, would @m1cr0man know more on this?

Reply:
It's a systemd service so logs will be captured like any other service on the box with appropriate tags and sent to syslog (currently influx so viewable in chronograf) :)